include/deal.II/lac/full_matrix.h

```diff
@@ -68,6 +70,12 @@ template <typename number>
 class FullMatrix : public Table<2, number>
 {
 public:
+  // The assertion in full_matrix.templates.h for whether or not a number is
+  // finite is not compatible for AD number types.
```

Comment:
probably ~finite is~?

Reply:
nevermind.

Reply:
The way that this is worded is correct. I'll agree though that it is a bit clumsy :-)

Reply:
> The way that this is worded is correct.

i know, i just had to read it more than once to get it right ;-)